webpack.config.js

```diff
@@ -80,7 +80,7 @@ const config = {
 		packageNames.reduce( ( memo, packageName ) => {
 			memo[ packageName ] = `./node_modules/@wordpress/${ packageName }`;
 			return memo;
-		}, {} ),
```

Comment:
Maybe we could avoid this change? Trailing commas are recommended in Gutenberg

Reply:
When a tried run "npm run dev" I got unexpected token ')' in line 84.

Reply:
It looks like you are using `node 6` which had a bug and the same thing was merged into master yesterday to fix it. Might be better to switch to `node 8`anyway as the recommended version.

#3757